packages/react-magma-dom/src/components/Input/index.tsx

```diff
@@ -3,7 +3,7 @@ import styled from '../../theme/styled';
 import { InputBase, InputBaseProps, InputSize } from '../InputBase';
 import { InputMessage } from './InputMessage';
 import { Label } from '../Label';
-import { useGenerateId } from '../../utils';
+import { useGenerateId, Omit } from '../../utils';
```

Comment:
is `Omit` being used?

Reply:
Yeah, it's in the props interface.  I noticed an error in that file, and it looked like importing `Omit` was what was missing.

Reply:
Yeah, it's in the props interface. I noticed an error in that file, and it looked like importing Omit was what was missing.